phantom/rand.sh

```
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+echo "Password (bcrypt):"
+printf '%s\n' $(openssl rand 256 | tr -dc '[:graph:]' | tr -d "\'\"\\\#\$" | head -c 72)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)